client/signup/steps/website-content/wordpress-media-upload.tsx

```diff
@@ -184,25 +214,28 @@ export function WordpressMediaUpload( {
 			);
 		case UPLOAD_STATES.FAILED:
 			return (
-				<FilePicker accept="image/*" onPick={ onPick } key={ mediaIndex }>
+				<>
 					<FileSelectThumbnailContainer>
-						<img src={ placeholder } alt="placeholder" />
+						<FileInput type="file" onChange={ onPick } accept={ allowedFileTypesString } />
+						{ /* TODO: Change placeholder to video for video media type */ }
```

Comment:
I'm guessing this TODO has been completed. 😄

Reply:
Actually want to fix the pre upload placeholder :D